src/components/link/link-button/link-button.spec.ts

```diff
@@ -12,14 +12,7 @@ describe(`sbb-link-button`, () => {
 
   beforeEach(async () => {
     element = await fixture(html`
-      <sbb-link-button
-        name="name"
-        type="button"
-        form="form"
-        value="value"
-        size="m"
-        aria-label="Travelcards &amp; tickets"
-      >
+      <sbb-link-button name="name" type="button" form="form" value="value" size="m">
```

Comment:
Old code has aria-label, is accessibility-label missing?

Reply:
The mistake was to remove `aria-label` (because link-button extends `ButtonBase`)